src/actions/course.js

```diff
@@ -32,6 +33,78 @@ export function createCourse(course) {
   }
 }
 
+/**
```

Comment:
I'd prefer if these functioned the same way that updating a queue works - doing a `PATCH` to `api/courses/:courseId` instead of a separate route per attribute. That'll match with our existing conventions and help prevent an explosion of actions/reducers/router handlers/etc as more settings are added.